std/haxe/io/MIME.hx

```diff
@@ -0,0 +1,476 @@
+package haxe.io;
+
+/**
+    Html MimeType Enum
+    @see : http://www.sitepoint.com/web-foundations/mime-types-complete-list/
+ **/
+
+@:enum abstract MIME(String) from String to String {
```

Comment:
This is more like a mapping of file extensions to the corresponding mime types.
Real enum should not have duplicated values.
I propose something like this for mime types:
```haxe 
enum abstract MimeTypes(String) from String to String {
  var XWorldX3Dmf = 'x-world/x-3dmf';
  var ApplicationOctetStream = 'application/octet-stream';
  var ApplicationXAuthorwareBin = 'application/x-authorware-bin';
  <...>
}
```

Reply:
I have a mapping in hxgithub: https://github.com/Simn/hxgithub/blob/master/src/github/helper/MediaTypesMap.hx

It's more practical to have this as a map because you oftentimes want to use it as a lookup.

Reply:
Depends on a use case.
As a server developer, I often use mime types without references to files.